src/components/Events/EventCard.js

```diff
@@ -0,0 +1,134 @@
+import React from 'react'
+import styled from 'styled-components'
+import breakpoint from 'styled-components-breakpoint'
+
+import { Row, Col } from '../grid'
+import StyledLink from '../Common/StyledLink'
+
+import { CardTitle, BodyPrimary, CalendarDay } from '../Typography'
+import remcalc from 'remcalc'
+
+const CalendarPage = styled.div`
+  padding: 24px;
+  border: solid;
+  border-color: ${props => props.theme.colors.border};
+  border-width: thin;
+  width: 108px;
+  height: 108px;
+  text-align: center;
+
+  ${breakpoint('tablet')`
+    width: 143px;
+    height: 143px;
+  `}
+`
+
+const CenteredBodyPrimary = styled(BodyPrimary)`
+  text-align: center;
+  line-height: ${remcalc(18)};
+
+  ${breakpoint('tablet')`
+    line-height: ${remcalc(24)};
+
+  `}
+`
+
+const NoPaddingCol = styled(Col)`
+  padding: 0;
+`
+
+const DateCard = ({ date }) => (
+  <CalendarPage>
+    <CalendarDay noPadding>{date.day}</CalendarDay>
+    <CenteredBodyPrimary noPadding>
+      {date.month.toUpperCase()}
+    </CenteredBodyPrimary>
+  </CalendarPage>
+)
+
+const AlignRightCol = styled(Col)`
+  ${breakpoint('tablet')`
+text-align: right;
+
+  `}
+`
+
+const SelectivePaddingCardTitle = styled(CardTitle)`
+  padding-top: 0;
+  padding-bottom: ${remcalc(6)};
+
+  ${breakpoint('tablet')`
+padding-top: 0;
+
+  `}
```

Comment:
All typography elements from common should not have their padding changed at all. They follow a defined set of line heights and paddings, when you start to change this and divert from the rules we have set out you start to break down the integrity of the sites typography.

Reply:
So is the correct approach to wrap this line of text in a div and give the div the padding as needed?

Reply:
yes